### PR TITLE
Clarify QDevice/QNetd configuration

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

I've attempted to make the descriptions of the QDevice/QNetd configuration a bit clearer.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1227649
* jsc#DOCTEAM-1497

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP6
  - [x] 15 SP5
  - [ ] 15 SP4
  - [ ] 15 SP3
  - [ ] 15 SP2
- SLE-HA 12
  - [ ] 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by tahliar** [comment]
image
image

File: xml/ha_qdevice-qnetd.xml [hunk 1]
```diff
@@ -264,6 +264,12 @@
          of your cluster, or you can add them later. This procedure documents how
          to add them later.
       </para>
+      <important role="compact">
+        <para>
+          To complete the &qnet; configuration, the clients must have SSH access
```
Comment (jirib) [inline xml/ha_qdevice-qnetd.xml:269]: this lovely crmsh needs root ssh iiuc

**Comment by jirib** [comment]
Can we get the manual - that is, without crmsh - steps? With parts from corosync.conf and covering qnetd and qdevice as well? If so, it would be great!

File: xml/ha_qdevice-qnetd.xml (hunk 1 above)
Comment (tahliar) [reply xml/ha_qdevice-qnetd.xml]: Thanks!

**Review by dariavladykina — COMMENTED** [review]
Hi, just a minor nit here. Thanks!

**Pull request merged**